node/src/models/Lesson.ts

```diff
@@ -22,16 +22,29 @@ export interface UpdateLessonExpectation {
   hints: UpdateHint[];
 }
 
+export interface UpdateMediaProp {
+  name: string;
+  value: string;
+}
+
+export interface UpdateMedia {
+  url: string;
+  type: string;
+  props: UpdateMediaProp[];
+}
+
 export interface UpdateLesson {
   lessonId: string;
   name: string;
   intro: string;
   dialogCategory: string;
   question: string;
-  image: string;
+  image?: string;
+  media: UpdateMedia;
+  learningFormat: string;
```

Comment:
it's probably easier for you if you make `learningFormat` optional too, and give it a default at gql resolve time. Otherwise you'll have to populate it in a migration for all existing lessons